include/cppflow/tensor.h

```diff
@@ -205,7 +207,10 @@ namespace cppflow {
 
         // Convert to correct type
         const auto T_data = static_cast<T*>(raw_data);
-        return std::vector<T>(T_data, T_data + size);
+        std::vector<T> r(T_data, T_data + size);
+        TF_DeleteTensor(res_tensor);
```

Comment:
This is not very compatible with your span PR, because we need to obtain the ownership of `res_tensor` and wrap its internal data pointer as `span`. Why not store it in `this->tf_tensor`?

Reply:
> This is not very compatible with your span PR, because we need to obtain the ownership of `res_tensor` and wrap its internal data pointer as `span`. Why not store it in `this->tf_tensor`?

I commented here: https://github.com/serizba/cppflow/pull/68#issuecomment-723402793

Reply:
I'll update this in #68 . This PR only for memory leak.